docs/tutorial.md

```diff
@@ -150,15 +156,14 @@ commit ID, which changes when you rewrite the commit. You can give either ID
 to commands that take revisions as arguments. We will generally prefer change
 IDs because they stay the same when the commit is rewritten.
 
-By default, `jj log` lists your local commits, with some remote commits added
-for context.  The `~` indicates that the commit has parents that are not
```

Comment:
The "with some remote commits added for context" was meant to be about the `ancestors(x, 2)` part (`orrkosyo` above), so that's still true

Reply:
OK, reverted. (I thought it referred to remote-tracking branches, as you probably guessed)